### ISSUE TITLE
Fenced code blocks in footnotes

### ISSUE THREAD

**Issue by charpov** [jgm/pandoc#3207]
Is there a rule against those?  LaTeX chokes on the output from Pandoc 1.18.

```
Is this supposed to work[^1] in Pandoc?

[^1]: foo

    ~~~java
    bar
    ~~~
```

**Comment by jgm** [comment]
They're okay as far as pandoc is concerned; it seems to
be a LaTeX specific issue.   You need `\VerbatimFootnotes`
in the preamble.

I thought pandoc was putting it there automatically,
but apparently not -- I'll look into this.

**Comment by jgm** [comment]
Workaround for you:  put `\VerbatimFootnotes` in your preamble (at the end, after the highlighting stuff has loaded).

Change needed to pandoc:  we need to ensure that `\VerbatimFootnotes` gets into the preamble after `fancyvrb` is loaded.  Currently this is done for non-highlighted code blocks but not for highlighted.

**Comment by jgm** [comment]
Fixed by 6e2019d58812df59d1cb74723c71305c9b233bae
